packages/amplify-e2e-tests/src/utils/pinpoint.ts

```diff
@@ -18,13 +18,42 @@ const settings = {
   pinpointResourceName: 'testpinpoint',
 };
 
+const defaultPinpointRegion = 'us-east-1';
+const serviceRegionMap = {
+  'us-east-1': 'us-east-1',
+  'us-east-2': 'us-east-1',
+  'sa-east-1': 'us-east-1',
+  'ca-central-1': 'us-east-1',
+  'us-west-1': 'us-west-2',
+  'us-west-2': 'us-west-2',
+  'cn-north-1': 'us-west-2',
+  'cn-northwest-1': 'us-west-2',
+  'ap-south-1': 'us-west-2',
+  'ap-northeast-3': 'us-west-2',
+  'ap-northeast-2': 'us-west-2',
+  'ap-southeast-1': 'us-west-2',
+  'ap-southeast-2': 'us-west-2',
+  'ap-northeast-1': 'us-west-2',
+  'eu-central-1': 'eu-central-1',
+  'eu-west-1': 'eu-west-1',
+  'eu-west-2': 'eu-west-1',
+  'eu-west-3': 'eu-west-1',
+};
+
+function mapServiceRegion(region) {
```

Comment:
can use [lodash get](https://lodash.com/docs/4.17.15#get) and specify the default value param instead of this function